compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1417,14 +1417,6 @@ rustc_queries! {
         separate_provide_extern
     }
 
-    /// Given a crate, look up all trait impls in that crate.
-    /// Return `(impl_id, self_ty)`.
-    query all_trait_implementations(_: CrateNum)
```

Comment:
I'm not super familiar with the rustc side of things, is the query just being removed due to being dead code?

Reply:
Yes, it was only used from rustdoc, and now it's no longer used.